### ISSUE TITLE
KMP iOS build fails with code 65

### ISSUE THREAD

**Issue by janesp** [Kotlin/KMP-App-Template#8]
The downloaded KMP-App-Template raises an error «Build iOS application: failed with code 65» (BTW - same with Android Studio App project template).

Environment:

- MacBook Pro 2021 (M1 Max)
- mocOS Sonoma 14.4.1
- xcode 15.3

kdoctor:

Environment diagnose (to see all details, use -v option):
[✓] Operation System
[✓] Java
[✓] Android Studio
[✓] Xcode
[✓] CocoaPods

Conclusion:
  ✓ Your operation system is ready for Kotlin Multiplatform Mobile Development!

Same before and after upgrading from Gradle 8.2.2 to Gradle 8.4.0

Not successful with [Stack Overflow article](https://stackoverflow.com/questions/77179518/build-failed-in-kmm-project-in-ios-emulator-due-to-xcode-run-script-phase-withou?rq=2)

Any hints would be much appreciated!

**Comment by zsmb13** [comment]
Glad you found a solution @janesp! What version of Java were you using that didn't work?

**Comment by janesp** [comment]
I was on Java 11 and upgraded to Java 22.

I have to admit that I am not a developer anymore (rather did this in my younger years), only occasionally trying out new solutions out of interest or for fun, thus my outdated Java environment.

Nevertheless I got KMP up and running within a day, just by following the tutorial videos and spending quite some research time to solve plugin and version issues.

BTW - We implemented KMP-style solutions in the 80s to run our Modula-2 software on GUIs of both ETH Lilith machines and early Macintoshs. That's why KMP has my sympathies 😊.

FYI - Here is the reason why we are looking into KMP - https://github.com/Abdagon/health-ssi-2
